subiquity/cmd/server.py

```diff
@@ -193,6 +193,7 @@ def main():
 
     logfiles = setup_logger(dir=logdir, base="subiquity-server")
 
+    logging.captureWarnings(True)
```

Comment:
Is there a need to capture the warnings in the server or just for consistency?

Reply:
I have to admit I have no idea where the warnings raised by the server go. I would say capturing them (so they end up in the logs) is the best option but <something> seems to drop them first :shrug: